kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -0,0 +1,200 @@
+oracle-db    NodePort    10.99.94.157   <none>        1521:30011/TCP   32m
+
+The DB Connection String is oracle-db.default.svc.cluster.local:1521/devpdb.k8s which can be used as  rcuDatabaseURL parameter to domain.input.yaml file while creating a Fusion Middleware domain in Operator Environment 
+
+The Database can be accessed thru external NodePort outside of Kubernates cluster using the URL String <hostmachine>:30011/devpdb.k8s
+
+Note : Domain-in-Image model need public DB url as rcuDatabaseURL parameter to configure Fusion Middleware domain in Operator Environment
+
+```
+
+```
+$ ./create-rcu-schema.sh -s <schemaPrefix> -d <dburl> 
+
+
+```
+The parameters are as follows:
+
+```  
+  -s  RCU Schema Prefix, Must be specified
+  -d  RCU Oracle Database URL (default oracle-db.default.svc.cluster.local:1521/devpdb.k8s) 
+```
+```
+The script Generates the RCU schema based Schema Prefix and RCU DB URL
+$ ./create-rcu-schema.sh -s domain1
+[oracle-db-756f9b99fd-r6ghb] already initialized .. 
+Checking Pod READY column for State [1/1]
+NAME                         READY   STATUS    RESTARTS   AGE
+oracle-db-756f9b99fd-r6ghb   1/1     Running   0          109s
+Trying to pull repository container-registry.oracle.com/middleware/fmw-infrastructure ... 
+12.2.1.3: Pulling from container-registry.oracle.com/middleware/fmw-infrastructure
+Digest: sha256:215d05d7543cc5d500eb213fa661753ae420d53e704baabeab89600827a61131
+Status: Image is up to date for container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3
+pod/rcu created
+[rcu] already initialized .. 
+Checking Pod READY column for State [1/1]
+Pod [rcu] Status is Ready Iter [1/60]
+NAME   READY   STATUS    RESTARTS   AGE
+rcu    1/1     Running   0          7s
+NAME                         READY   STATUS    RESTARTS   AGE
+oracle-db-756f9b99fd-r6ghb   1/1     Running   0          2m2s
+rcu                          1/1     Running   0          12s
+CLASSPATH=/usr/java/jdk1.8.0_211/lib/tools.jar:/u01/oracle/wlserver/modules/features/wlst.wls.classpath.jar:
+
+PATH=/u01/oracle/wlserver/server/bin:/u01/oracle/wlserver/../oracle_common/modules/thirdparty/org.apache.ant/1.9.8.0.0/apache-ant-1.9.8/bin:/usr/java/jdk1.8.0_211/jre/bin:/usr/java/jdk1.8.0_211/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/java/default/bin:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:/u01/oracle/container-scripts:/u01/oracle/wlserver/../oracle_common/modules/org.apache.maven_3.2.5/bin
+
+Your environment has been set.
+Check if the DB Service is Ready to accept Connection ?
+DB Connection URL [oracle-db.default.svc.cluster.local:1521/devpdb.k8s] and schemaPrefix is [domain1]
+[1/20] Retrying the DB Connection ...
+[2/20] Retrying the DB Connection ...
+[3/20] Retrying the DB Connection ...
+
+**** Success!!! ****
+
+You can connect to the database in your app using:
+
+  java.util.Properties props = new java.util.Properties();
+  props.put("user", "scott");
```

Comment:
these credentials have to be wrong, and we should not be printing them out anyway